secant/Util/RecoveryPhraseStorage.swift

```diff
@@ -0,0 +1,228 @@
+//
+//  RecoveryPhraseStorage.swift
+//  secant-testnet
```

Comment:
I think we can delete the line `secant-testnet`

Reply:
I think we need some general approach to this. There are 44 instances of `secant-testnet` in the project. The project itself is called secant-testnet, any contributor forking the repo will see secant-testnet xcode project, opens secant-testnet project, anytime when contributor creates a swift file, this line will be there automatically in the header. Removing it here is not going to do any meaningful work. If we won't do deal with it, we need separate ticket that is proposing general solution and deals with the future as well. I see several steps:
1. rename the project
2. refactor all secant-testnet instances
3. commit
4. any contributor/internal dev from that moment on will have right project name in the header and there will be no secant-testnet anymore

Reply:
I created this issue. 
https://github.com/zcash/secant-ios-wallet/issues/207